### PR TITLE
push docker containers with tagged version release

### PR DESCRIPTION
**Pull request by szuecs**
Signed-off-by: Sandor Szücs <sandor.szuecs@zalando.de>

### CODE REVIEW

File: delivery.yaml [hunk 1]
```diff
@@ -19,10 +19,12 @@ pipeline:
     cmd: |
       if [[ $CDP_TARGET_BRANCH == master && ! $CDP_PULL_REQUEST_NUMBER ]]; then
         IMAGE=registry-write.opensource.zalan.do/teapot/stackset-controller
+        VERSION=$(git describe --tags --always --dirty)
```
Comment (aermakov-zalando) [inline delivery.yaml]: Please remove `--dirty`.

Comment (aermakov-zalando) [reply delivery.yaml]: I'd also force go to have go.mod as readonly to avoid surprises, but it's not strictly necessary.

Comment (szuecs) [reply delivery.yaml]: done

**Comment by szuecs** [comment]
:+1:

**Pull request merged**